javascript/ql/test/library-tests/InterProceduralFlow/TaintTracking.expected

```diff
@@ -36,6 +36,9 @@
 | tst2.js:3:17:3:26 | "tainted2" | tst2.js:11:15:11:24 | g(source2) |
 | tst2.js:6:24:6:37 | "also tainted" | tst2.js:10:15:10:24 | g(source1) |
 | tst2.js:6:24:6:37 | "also tainted" | tst2.js:11:15:11:24 | g(source2) |
+| tst4.js:2:16:2:24 | "tainted" | tst4.js:15:15:15:31 | id(still_tainted) |
```

Comment:
just checking: these tests behave identically on the current `ql/master`, right?

Reply:
I think so, yes. This is a regression test for a mistake I made while working on this PR but later fixed.